runtimes/runtimes/standalone.ts

```diff
@@ -255,3 +269,9 @@ export const standalone = (props: RuntimeProps) => {
         lspConnection.listen()
     }
 }
+
+async function encryptRequest(request: Object, key: string): Promise<string> {
```

Comment:
nit: maybe better to move this function to separate util file, or to https://github.com/aws/language-server-runtimes/blob/main/runtimes/runtimes/auth/standalone/encryption.ts to keep helper functions related to encryption in single place